contents/blog/google-optimize-alternatives.md

```diff
@@ -0,0 +1,199 @@
+---
+date: 2023-01-23
+title: The best Google Optimize alternatives for apps and websites
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - andy-vandervell
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+category: General
+tags:
+  - Guides
+  - Comparisons
+---
+
+Google recently confirmed it is sunsetting Google Optimize, its A/B testing platform. In a [statement](https://support.google.com/optimize/answer/12979939) on its website, Google said:
+
+> "Google Optimize and Optimize 360 will no longer be available after **September 30, 2023**. Your experiments and personalizations can continue to run until that date. Any experiments and personalizations still active on that date will end."
+
+The statement went on to say Google is working on adding A/B testing functionality to Google Analytics 4, its somewhat divisive replacement for Google Analytics 360 (Universal Analytics). But Google **didn't commit to a launch date** for this A/B testing functionality, which suggests it may not arrive before Optimize is closed.
+
+If this sounds like bad news to you, consider one of following six Google Optimize alternatives instead. 
+
+> **How did choose these tools?** There are dozens of A/B testing platforms, so we tried to include a cross-section of tools designed for engineers, marketing teams, and different use cases. We prioritized tools that are self-serve (i.e. no demo or quote needed to open an account) and/or have transparent pricing.
+
+## 1. PostHog
+
+- **Free tier:** Yes
+- **Transparent pricing:** Yes
+- **Unlimited experiments:** Yes
+- **A/B and multivariate tests:** Yes
+- **Session recording:** Yes
+- **Product analytics:** Yes
+- **Feature flags:** Yes
+- **Multi-armed bandit:** No
+- **Visual editor:** No
+- **Landing page builder:** No
+- **Best for:** Product and engineering teams
+
+[PostHog](posthog.com/) is a comprehensive Google Optimize and [GA4 alternative](/blog/ga4-alternatives) that integrates an A/B and multivariate testing suite with event-based analytics, session recording, and feature flags.
+
+PostHog experiments can be targeted by geography, cohort, and user property. It also supports minimum goals for success, secondary goals, and estimates how long it will take to get a statistically significant result. PostHog's experimentation suite is built on its feature flag functionality, making it easy to rollout winning variants.
+
+As an all-in-one platform, PostHog simplifies your data and analytics stack by replacing multiple tools. Tight integration also makes it easy to quickly create experiments using existing cohorts in your product analytics, rather than connecting an external data source.
```

Comment:
```suggestion
As an all-in-one platform, PostHog simplifies your data and analytics stack by replacing multiple tools. This makes it easy to quickly create experiments using existing data and cohorts in your product analytics, rather than connecting an external data source.
```